docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
+|---|---|---|
+| `int_exp` | Required `int` value or expression to be converted to a `timestamp`. | `int` |
+| `timeunit` | Optional `string` value specifying the granularity of the input `int_exp`, defaults to `s`. see [TIMEUNIT values](#timeunit-value).| `string` |
+
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `timestamp` | Returns timestamp equivalent of `int_exp`, it converts the input using unix epoch as base and the optional `timeunit` to determine the granularity of the input value. |
+
+## Remarks
+If `timeunit` is not supplied, the granularity of the input will default to seconds.
+
+### TIMEUNIT value
```

Comment:
Convert this information to an `/_include` file under `/sql-guide`

Proposed title: timestamp-timeunit-table.md

Content:

```
### TIMEUNIT value

| Unit | Declaration |
|---|---|
| seconds (default) | `s` |
| milliseconds | `ms` |
| microseconds | `us` |
| nanoseconds | `ns` |
```

Insert into `/data-type-timestamp.md` and `/function-totimestamp.md` with this shortcode:

```
{% include /sql-guide/timestamp-timeunit-table.md %}